test/unit/rules/lint-no-positive-tabindex-test.js

```diff
@@ -14,6 +14,9 @@ generateRuleTests({
     '<span tabindex={{"-1"}}>baz</span>',
```

Comment:
Ideally these would be buttons and not spans. Spans are not interactive elements.

Reply:
Great point! Updated during pairing session with @MelSumner...